be-typescript-express/add-to-package-json.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python2
+
+import sys
+import json
+
+def appendCommandsToPackageJson(packageJson):
+    with open(packageJson, 'r') as file:
+        package = file.read()
+
+    obj = json.loads(package)
+    obj['scripts'].update(
+            {
+                "build": "tsc --watch",
+                "start": "node dist/src/index.js",
+                "test": "JEST_JUNIT_OUTPUT_DIR='build/test-results/test' JEST_JUNIT_OUTPUT_NAME='test-results.xml' npx jest --reporters=default --reporters=jest-junit --coverage --coverageDirectory=coverage_output --forceExit ./dist"
+            }
+        )
+
+    with open(packageJson, 'w') as file:
+        file.write(json.dumps(obj, indent=4, sort_keys=True))
+        file.close()
+
+if __name__ == "__main__":
+    appendCommandsToPackageJson(sys.argv[1])
```

Comment:
I think this could be done in Groovy, right from the `Jenkinsfile`. Might be a bit easier as we only have Groovy code then, not Groovy and Python ... What do you think?

Reply:
I essentially wanted to keep the Jenkinsfile short and hence we ended up with an `init.sh` and a python file. The code in both scripts is easy to understand but I get the upside of having the logic written in Groovy. If a longer Jenkinsfile is ok for you, then sure

Reply:
@michaelsauter Can't read a file with method without script approval unfortunately. 
`Scripts not permitted to use new java.io.File java.lang.String`
Can we keep the python script? It's also easier for me to test and maintain it as is.

Reply:
Sure we can keep it. It should be very easy to read the JSON in Groovy though: https://www.jenkins.io/doc/pipeline/steps/pipeline-utility-steps/#readjson-read-json-from-files-in-the-workspace. Then you simply add the key, and use `writeFile` to write. But this was just a suggestion, I don't mind either way.

Reply:
Ah nice, didn't know that read/write JSON methods existed in Jenkins. It works now.